File: lockoutGoal.java
public class lockoutGoal {
    private String type;
    private int value;
    private int apreward;
    private boolean done;
    public lockoutGoal(String t, int v, int apr) {
        type = t;
        value = v;
        apreward = apr;
        done = false;
    }
    //GETTERS
    public String getType() {return type;}
    public int getValue() {return value;}
    public int getAPR() {return apreward;}
    public boolean getDone() {return done;}
    //SETTERS
    public void setDone(boolean b) {done = b;}
}

File: game/lockoutGoal.js
export class lockoutGoal {
    type;
    value;
    apreward;
    done;
    constructor(type, value, apreward) {
        this.type = type;
        this.value = value;
        this.apreward = apreward;
        this.done = false;
    }
    //GETTERS
    getType() {return type;}
    getValue() {return value;}
    getAPR() {return apreward;}
    getDone() {return done;}
    //SETTERS
    setDone(b) {done = b;}
}